libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -227,9 +232,9 @@ bool VoteManager::addVerifiedVote(const std::shared_ptr<Vote>& vote) {
     LOG(log_nf_) << "Added verified vote: " << hash;
     LOG(log_dg_) << "Added verified vote: " << *vote;
 
-    // Save in db only those reward votes that have the same round as round during which we pushed the block into chain
-    if (is_valid_potential_reward_vote && reward_votes_round_ == vote->getRound()) {
+    if (is_valid_potential_reward_vote) {
       db_->saveRewardVote(vote);
```

Comment:
we could also call this saveExtraRewardVote as well as db column to show that not all reward votes are store here as previously

Reply:
Renamed everything related to extra rewards